CMakeLists.txt

```diff
@@ -58,10 +58,6 @@
 
 cmake_minimum_required(VERSION 3.16.0)
 
-if(POLICY CMP0076)
-  cmake_policy(SET CMP0076 NEW)
-endif()
```

Comment:
what is the purpose of this?

Reply:
hmm that slept in, with 3.16 that policy is already set to new so not needed anymore...